pkg/ottl/ottlfuncs/README.md

```diff
@@ -1335,6 +1335,132 @@ Examples:
 - `ParseKeyValue("k1!v1_k2!v2_k3!v3", "!", "_")`
 - `ParseKeyValue(attributes["pairs"])`
 
+### ParseSimplifiedXML
+
+`ParseSimplifiedXML(target)`
+
+The `ParseSimplifiedXML` Converter returns a `pcommon.Map` struct that is the result of parsing the target string without preservation of attributes or extraneous text content.
+
+The goal of this Converter is to produce a more user-friendly representation of XML data than the `ParseXML` Converter,
+which produces a verbose *encoding* of XML data.
```

Comment:
Adding a section for when to use this converter instead of ParseXML would be helpful

Reply:
I expanded on this section a bit.